solutions/usaco-1228.mdx

```diff
@@ -83,4 +84,57 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+
+```python
+import sys
+t = sys.stdin.readline
+
+def solution():
+
+	n = int(t())
+	
+	g, l, ans = [], [], float('inf')
+	
+	for _ in range(n):
+	
+		a, b = t().split()
+
+		'''
+		The position is stored in g if the cow claims that Bessie is hiding at a location greater than b.
+		The position is stored in l if the cow claims that Bessie is hiding at a location less than b.
+		'''
+
+        	if a == "G": g.append(int(b))
+		else: l.append(int(b))
+        
+	bessie = g + l + [l[0] - 1] + [g[-1] + 1]
```

Comment:
what the

Reply:
its correct? should i add comment above 

# considers all cases including when Bessie's location is greater than the greatest guess among  the "greater" guesses and when Bessie's location is less than the least guess among "less than" guesses ?